tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/temporal/utils.scala

```diff
@@ -35,6 +37,22 @@ case class PredExs(
 
 package object utils {
   val builder: ScopedBuilder = new ScopedBuilder()
+
+  /* replaces all occurences of := with =.
+  For example, x' := foo becomes x' = foo. */
+  def rewriteAssignmentsAsEquality(ex: TlaEx): TlaEx = {
```

Comment:
If this is a generic utility it needs a `LetInEx` case.

Reply:
I wanted to use this only where it's assumed that there are no let-in expressions, but maybe it's better to just add a case for it

Reply:
Judging from the functionality here, I'd just move this to `lir/transformations/standard/` as a transformation and see how the other txs are implemented.